html.md

```diff
@@ -285,7 +285,7 @@ echo Form::select('size', array('L' => 'Large', 'S' => 'Small'));
 #### Generating A Drop-Down List That Allows Multiple Selection
 
 ```php
-echo Form::select('size', array('L' => 'Large', 'S' => 'Small'), null, ['multiple' => true]);
+echo Form::select('size[]', array('L' => 'Large', 'S' => 'Small'), null, ['multiple' => 'multiple']);
```

Comment:
Actually, merely having `['multiple']` is sufficient. This is also true for any other boolean HTML5 attributes like `disabled` or `readonly`. 

```php
>>> echo Form::select('name', [], null, ['multiple'])
<select multiple="multiple" name="name"></select>⏎
```